prow/cluster/monitoring/mixins/prometheus/boskos_alerts.libsonnet

```diff
@@ -0,0 +1,34 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'Boskos resource usage',
+        rules: [
+          {
+            alert: 'Low resource availability (<10% free)',
+            // This expression calculates the percentage of resources of each type that are free.
+            // If there are multiple instances the most pesimistic value is used.
+            // The threshold for the alert is <10% free.
+            // Resource pools with <= 5 resources are ignored since it is often expected for
+            // small pools to use 100% capacity.
```

Comment:
Should there be a separate alert for small resource pools possible (like 0 free resources for X time?)

Not blocking for this alert, just curious.

Reply:
That sounds reasonable if we could pick a reasonable value for X, but I'd imagine that the time a resource is held for is pretty dependent on the resource type. We could probably set a large upper bound, but it might need to be a day or more.
We can add another alert for this in a follow up if we get a better feel for what X should be or if it becomes necessary.

Reply:
Makes sense